superduperdb/jobs/job.py

```diff
@@ -246,7 +247,7 @@ def __call__(
         if not distributed:
```

Comment:
you see this one is important

```
if not distributed 
else
```
is not needed 

compute.submit() will send to dask if set DaskCOmpute else Local 

so in this file 

please remove run on dask and run on local 

also please remove distributed from the entire project

Reply:
@blythed can attest here

Reply:
Everything breaks If i change it.

Reply:
but it is needed may be I can file a pr
else there is no point of DaskCompute and LocalCompute abstraction